accelerator/common_core.c

```diff
@@ -349,7 +349,11 @@ status_t ta_find_transaction_objects(const iota_client_service_t* const service,
                            NUM_TRITS_SERIALIZED_TRANSACTION, NUM_TRITS_SERIALIZED_TRANSACTION);
       ret = cache_set(txn_hash, cache_value);
       if (ret != SC_OK) {
-        goto done;
+        if (ret == SC_CACHE_OFF) {
```

Comment:
Maybe we can modify the return value from cache, return SC_OK also if cache is not enable.
Then we can keep this branch condition like:
```
if (ret != SC_OK) {
   goto done;
}
```

Reply:
@wusyong how do you think

Reply:
I'm fine with the suggestion. We will have refactor cache layer in the future anyways.
```suggestion
        if (ret != SC_OK) {
          goto done;
        }
```

Reply:
I find a problem that we need to know if caching server is turned on, and it would be easier if we return an error status code

Reply:
I see. Just keep this then.